### ISSUE TITLE
[bug] BG2 'Badspells' - Tenser's Transformation

### ISSUE THREAD

**Issue by caillean7** [gemrb/gemrb#486]
See #465 

**Description**

> tenser's tranformation also disables innates (has the effects). We do not differentiate non-spell/magic innates yet (could be just a prefix thing: spin vs spcl).

Tenser's Transformation indeed disables only non-magical innates like Set Snare (SPCL412) or Slayer Change (SPIN822) on the original. So this needs implementing, those abilites currently work on GemRB. But it's probably no mere prefix thing as you can see from the above examples.

Some visual nitpickings just to point out any differences between the engines:

1. When clicking the Spell Casting button with TT active, it looks like this on IE:

![tenser_IE](https://user-images.githubusercontent.com/54738589/70616958-e462f880-1c0f-11ea-8a83-900917781def.jpg)

GemRB has the spells framed green, not grayed out:
![tenser_GemRB](https://user-images.githubusercontent.com/54738589/70617125-2e4bde80-1c10-11ea-90ef-506a234d3547.jpg)

2. With TT active the non-working innate spells could be grayed out. IE doesn't do it, they can still be selected, just the character won't do anything. It feels like they overlooked this, so maybe you'd consider deactivating those buttons as an improvement over the IE.

**Comment by lynxlynxlynx** [comment]
Thanks, we'll see. For the differentiation, we can probably use the spell's extended header field for ability type: 0/weapon/spell/item/ability.

**Comment by caillean7** [comment]
Do you need a list of all innate abilities that need deactivating for TT or are you all clear about this?

**Comment by lynxlynxlynx** [comment]
Nah, that'd be hundreds to check. Much easier with iesh, but the two you mentioned already match the type==ability criterion, so that's what we'll go with.
